dashboard/lib/policies/child_account.rb

```diff
@@ -27,6 +31,12 @@ def self.compliant?(user)
     user.child_account_compliance_state == ComplianceState::PERMISSION_GRANTED
   end
 
+  # Checks if a user is affected by a state policy but was created prior to the
+  # policy going into effect.
+  def self.grandfathered_user?(user)
```

Comment:
Thought about this one and ended up on `user_predates_policy?` to be the most explicit about what the function is doing. This felt the most generic, since depending on the policy, temporary/"at risk" might not be accurate. 

I added the comparison between start_date and current_date into the `parent_permission_required?` function since it felt like it fits there better.